### ISSUE TITLE
Flaky test for connections

### ISSUE THREAD

**Issue by cmdcolin** [GMOD/jbrowse-components#3623]
Seen https://github.com/GMOD/jbrowse-components/actions/runs/4612369000/jobs/8153187790#step:5:1539

This may be real as I have seen it fail locally also

**Comment by cmdcolin** [comment]
See also https://github.com/GMOD/jbrowse-components/pull/3502/files#diff-2e65875ef77d0f381fba98157b8aa25b2da0ef517bb711d13e949387f7507729 this refactoring to the test made on the react 18 branch may help fix

**Comment by cmdcolin** [comment]
again here https://github.com/GMOD/jbrowse-components/actions/runs/4621749069/jobs/8173556988

**Comment by rbuels** [comment]
yeah i've seen it locally too

**Comment by cmdcolin** [comment]
could optimistically close, if re-appears can re-open

**Comment by rbuels** [comment]
it is appearing in my PR #3629

**Comment by cmdcolin** [comment]
not sure if your branch is up to date with main

**Comment by cmdcolin** [comment]
context https://github.com/GMOD/jbrowse-components/pull/3630 

this pr didnt get linked here before, but hopefully may help

**Comment by rbuels** [comment]
This seems to have been introduced by 2d419f08d8aff329aa08568b0aba7cac0ae326da from #3625

**Comment by cmdcolin** [comment]
I don't believe it should be flaky if you merge master, it was fixed in https://github.com/GMOD/jbrowse-components/pull/3630. I addressed a bug that #3625 introduced (the Array.isArray... stuff was not parenthesized correctly, and I just converted it to an assemblyManager.get instead)

**Comment by rbuels** [comment]
Yeah you are right it's fixed on main
